src/core/lib/security/security_connector/ssl_utils.cc

```diff
@@ -41,12 +41,17 @@
 
 /* -- Constants. -- */
 
+#if defined(__Fuchsia__)
```

Comment:
Instead of checking Fuchsia here, can you define `GRPC_ROOT_PEM_PATH' in `include/grpc/impl/codegen/port_platform.h` for Fuchsia? We don't usually directly use #if <platform> in the code.

Reply:
done!